package.json

```diff
@@ -29,7 +29,7 @@
     "tape": "^4.6.0"
   },
   "dependencies": {
-    "iconutil": "^1.0.1",
+    "electron": "^1.4.15",
```

Comment:
also, I believe these dependencies need to be in `app/package.json` it looks like this was based off a branch I started though, so that's my mistake.